ci-operator/step-registry/redhat-developer/rhdh/nightly/redhat-developer-rhdh-nightly-commands.sh

```diff
@@ -52,7 +52,7 @@ GITHUB_ORG_NAME="redhat-developer"
 GITHUB_REPOSITORY_NAME="rhdh"
 
 export QUAY_REPO RELEASE_BRANCH_NAME
-QUAY_REPO="rhdh-community/rhdh"
```

Comment:
If we switch all the nightly jobs to `rhdh/rhdh-hub-rhel9` image, the _CI test triage_ can miss when the PR checks are broken, since they use the `rhdh-community/rhdh` image. One can be broken while the other can be running just fine, as we've seen in the past.
Sure, he should be checking https://prow.ci.openshift.org/?job=pull-ci-redhat-developer-rhdh-main-e2e-tests if PR checks are passing, but it might be worth having another job, that would be running with the `rhdh-community/rhdh` image. 
I think we should consider that.